From f70eb50b7107ee4e18cec3561bbdde1cbd0a0bdb Mon Sep 17 00:00:00 2001
From: David Gibson <dgibson@redhat.com>
Date: Mon, 27 Jul 2020 13:29:01 +0200
Subject: RHEL-only: Enable vTPM for POWER in downstream configs

RH-Author: David Gibson <dgibson@redhat.com>
Message-id: <20200522032718.387731-1-dgibson@redhat.com>
Patchwork-id: 96743
O-Subject: [RHEL-AV-8.3.0 qemu-kvm PATCH] RHEL-only: Enable vTPM for POWER in downstream configs
Bugzilla: 1781911
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: Greg Kurz <gkurz@redhat.com>

From: David Gibson <david@gibson.dropbear.id.au>

With the rebase to 5.0 we have the necessary code to implement virtual TPMs
for POWER targets.  However, it's not enabled in the Red Hat configuration.
This downstream only patch corrects that.

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1781911
Brew: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=28742072
Testing: With brewed qemu was able to see a vTPM device in a guest

Signed-off-by: David Gibson <dgibson@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 default-configs/ppc64-rh-devices.mak | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/default-configs/ppc64-rh-devices.mak b/default-configs/ppc64-rh-devices.mak
index ecbe53fe63..032422e103 100644
--- a/default-configs/ppc64-rh-devices.mak
+++ b/default-configs/ppc64-rh-devices.mak
@@ -32,3 +32,6 @@ CONFIG_XICS_SPAPR=y
 CONFIG_XIVE=y
 CONFIG_XIVE_SPAPR=y
 CONFIG_XIVE_KVM=y
+CONFIG_TPM_SPAPR=y
+CONFIG_TPM_EMULATOR=y
+CONFIG_TPM_PASSTHROUGH=y
-- 
2.27.0

